scalajslib/test/src/mill/scalajslib/NodeJSConfigTests.scala

```diff
@@ -0,0 +1,118 @@
+package mill.scalajslib
+
+import ammonite.ops._
+import mill._
+import mill.define.Discover
+import mill.eval.Evaluator
+import mill.scalalib.{CrossScalaModule, DepSyntax}
+import mill.util.{TestEvaluator, TestUtil}
+import utest._
+
+
+object NodeJSConfigTests extends TestSuite {
+  val workspacePath =  TestUtil.getOutPathStatic() / "hello-js-world"
+  val scalaVersion = "2.12.4"
+  val scalaJSVersion = "0.6.22"
+  val utestVersion = "0.6.3"
+  val scalaTestVersion = "3.0.4"
+  val nodeArg2G = "--max-old-space-size=2048"
+
+  trait HelloJSWorldModule extends CrossScalaModule with ScalaJSModule {
+    override def millSourcePath = workspacePath
+    def publishVersion = "0.0.1-SNAPSHOT"
+    override def mainClass = Some("Main")
+  }
+
+  object HelloJSWorld extends TestUtil.BaseModule {
+    val matrix = for {
+      scala <- Seq(scalaVersion)
+      nodeArgs <- Seq(Seq(), Seq(nodeArg2G))
+    } yield (scala, nodeArgs)
+
+    object helloJsWorld extends Cross[BuildModule](matrix:_*)
+    class BuildModule(val crossScalaVersion: String, nodeArgs: List[String]) extends HelloJSWorldModule {
+      override def artifactName = "hello-js-world"
+      def scalaJSVersion = NodeJSConfigTests.scalaJSVersion
+      override def nodeJSConfig = T { NodeJSConfig(args = nodeArgs) }
+    }
+
+    object buildUTest extends Cross[BuildModuleUtest](matrix:_*)
```

Comment:
As it is dedicated spec for node js configuration, I think we don't need utest/scalatest builds. We'll be good with one test framework(utest) and 4 test cases:
```
test
test2G
run
run2G
```

Reply:
This will simplify build configuration and tests too.